packages/react-app/src/components/quest.tsx

```diff
@@ -257,21 +260,27 @@ export default function Quest({
       <NoPaddingSplitStyled
         primary={
           <Outset gu16 className="pb-0">
+            {isEdit && <TitleStyled>Create Quest</TitleStyled>}
             <Outset gu8 vertical className="block">
               <NoPaddingSplitStyled
                 primary={
-                  <TextFieldInput
-                    id="title"
-                    label={isEdit ? 'Title' : undefined}
-                    isEdit={isEdit}
-                    isLoading={loading}
-                    placeHolder="Quest title"
-                    value={values.title}
-                    onChange={handleChange}
-                    fontSize="24px"
-                    tooltip="Title of your quest"
-                    wide
-                  />
+                  // <LinkStyled to={`/${ENUM_PAGES.Detail}?id=${values.address}`}>
+                  //   <Button icon={<IconPlus />} label="Details" wide mode="strong" />
+                  // </LinkStyled>
+                  <LinkStyled to={`/${ENUM_PAGES.Detail}?id=${values.address}`}>
```

Comment:
Only have a link on the summary mode, not in details mode nor create mode

Reply:
![image](https://user-images.githubusercontent.com/13890618/149414545-24803f2e-cafa-42ed-9621-f15a51124e76.png)